src/diffusers/models/unet_2d_blocks_flax.py

```diff
@@ -33,7 +33,7 @@ class FlaxCrossAttnDownBlock2D(nn.Module):
             Dropout rate
         num_layers (:obj:`int`, *optional*, defaults to 1):
             Number of attention blocks layers
-        attn_num_head_channels (:obj:`int`, *optional*, defaults to 1):
+        attn_num_heads (:obj:`int`, *optional*, defaults to 1):
```

Comment:
Let's avoid using two different variable names for the same purpose (`num_attn_heads` and `attn_num_heads`).